app/models/pawn.rb

```diff
@@ -1,3 +1,44 @@
 class Pawn < Piece
 
+	def valid_move?(new_x, new_y)
+		[move_number_allowed?(new_y), no_vertical_obstacle?(new_y), not_a_backwards_move?(new_y), diagonal_move_allowed?(new_x, new_y)].all?
```

Comment:
Thanks for that tip... I need to do some reading up on Ruby style guide :+1: I will make this change!